Pop-Grupa-A-Frontend/src/components/computationCockpit/taskList/TaskList.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+
+class TaskList extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            listitems: [],
+            userId: 1,
+            isLoading: false
+        };
+    }
+
+    getTasks() {
+        fetch("http://localhost:5000/task/" + this.state.userId, {
+            crossDomain: true,
+            method: 'get',
+        })
+            .then(response => response.json())
+            .then(response => {
+                this.setState({
+                    listitems: response
+                })
+            });
+    }
+
+    componentDidMount() {
+        this.getTasks();
+    }
+
+    componentWillUpdate() {
+        this.getTasks();
+    }
+
+    render() {
+        return (
+            <React.Fragment>
```

Comment:
React Fragment używajcie, jeśli w renderze macie co najmniej zwracacie dwa divy na tym samym poziomie zagłębienia.
Tutaj jest to niepotrzebne ,gdyż wspólnym rodzicem jest `<div className="row p-1">`